Dockerfile

```diff
@@ -19,24 +19,26 @@ COPY version/ version/
 ARG TARGETARCH
 ARG VERSION
 ARG GITVERSION
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on \
-    go build -a -ldflags "-X github.com/oam-dev/kubevela/version.VelaVersion=${VERSION:-undefined} -X github.com/oam-dev/kubevela/version.GitRevision=${GITVERSION:-undefined}" \
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} \
+    go build -a -ldflags "-s -w -X github.com/oam-dev/kubevela/version.VelaVersion=${VERSION:-undefined} -X github.com/oam-dev/kubevela/version.GitRevision=${GITVERSION:-undefined}" \
     -o manager-${TARGETARCH} main.go
 
-# Use ubuntu as base image for convenience.
+# Use alpine as base image to reduce image size
```

Comment:
@chengshiwen I think not only consider the image size, we can open the issue for discuss the choice of base image base on size, security and propularity.

Reply:
I just optimize this work with best practices, since in many docker libraries, debian and alpine are more popular, while images based on ubuntu are rare. As for security, no images will be safe and reliable. It doesn’t seem to make sense to only consider whether the base image is security.

Reply:
> I just optimize this work with best practices, since in many docker libraries, debian and alpine are more popular, while images based on ubuntu are rare. As for security, no images will be safe and reliable. It doesn’t seem to make sense to only consider whether the base image is security.

would you add the more issue annotation for the discussion not only for the reason for the `to reduce image size`, I think this discussion will help the other.

Reply:
OK

Reply:
See #1448, done